hip_data_tools/etl/google_sheet_to_athena.py

```diff
@@ -32,6 +33,11 @@ class GoogleSheetsToAthenaSettings:
         table_name: name of the athena table (eg: 'sheet_table')
         fields: list of sheet field names and types. Field names cannot contain hyphens('-')
```

Comment:
sounds like we need to put checks in place if this creates undesirable behaviour in Athena

Reply:
Fixed